daemon/Cargo.toml

```diff
@@ -44,3 +44,22 @@ rust-ini = "0.21.1"
 assert_matches = "1.5"
 serial_test = "3.1.1"
 tracing-test = "0.2.4"
+
+[profile.release]
+# Stripping symbols (not debuginfo) reduces binary size by ~1.5x.
```

Comment:
```suggestion
# Stripping symbols (not debuginfo) reduces binary size by ~1.5x.
# This is the default profile for `cargo install`:
# https://doc.rust-lang.org/cargo/reference/profiles.html#profile-selection.
# TODO: We may want to make [profile.release-size] the default
# once we're out of the fast iteration development phase.
[profile.release]
```

Reply:
I personally think for *installation* it would later be acceptable that it's building a bit slower, but also agree with you that `--release` is the easiest to type, so let's keep it like this for now :)